### PR TITLE
[ADD] stock_by_warehouse: add context for multi-warehouse

### PR DESCRIPTION
**Pull request by GavyMG**
For example: when we want to show only the sale warehouse and its resupply warehouses on the sale order line

https://git.vauxoo.com/absa/absa/-/merge_requests/1326

### CODE REVIEW

File: stock_by_warehouse/models/product_product.py [hunk 1]
```diff
@@ -77,11 +77,20 @@ def _compute_get_quantity_warehouses_json(self):
         # itself, we enable this context management
         warehouse_id = self._context.get('warehouse_id')
 
-        for warehouse in self.env['stock.warehouse'].sudo().search([]):
+        # Just in case it's asked from only a group of warehouses
+        warehouse_ids = self._context.get('warehouse_ids', False)
+        some_wh = False
```
Comment (luisg123v) [inline stock_by_warehouse/models/product_product.py:82]: This variable is not necessary, is it?

File: stock_by_warehouse/models/product_product.py [hunk 2]
```diff
@@ -77,11 +77,20 @@ def _compute_get_quantity_warehouses_json(self):
         # itself, we enable this context management
         warehouse_id = self._context.get('warehouse_id')
 
-        for warehouse in self.env['stock.warehouse'].sudo().search([]):
+        # Just in case it's asked from only a group of warehouses
+        warehouse_ids = self._context.get('warehouse_ids', False)
```
Comment (luisg123v) [inline stock_by_warehouse/models/product_product.py]:
```suggestion
        warehouse_ids = self._context.get('warehouse_ids')
```

**Review by luisg123v — CHANGES_REQUESTED** [review]

File: stock_by_warehouse/models/product_product.py (hunk 1 above)
Comment (GavyMG) [reply stock_by_warehouse/models/product_product.py:82]: To avoid comparing if not warehouse_id and warehouse_ids in every iteration.

**Comment by moylop260** [comment]
@JulioSerna 

Could you check it, please'

File: stock_by_warehouse/models/product_product.py (hunk 2 above)
Comment (GavyMG) [reply stock_by_warehouse/models/product_product.py]: @luisg123v 

could we merge this?? (You can commit the suggestion)

**Review by luisg123v — APPROVED** [review]

**Pull request merged**